Python: Remove usernames as sensitive source for cleartext queries (#7652)
github/codeql: python/ql/src/change-notes/2021-01-19-remove-cleartext-fps.md

---
category: majorAnalysis
---

> * No longer consider usernames or other account information as sensitive data for the queries `py/clear-text-logging-sensitive-data` and `py/clear-text-storage-sensitive-data`, since this has lead to many false positives.

Reviewer: ```suggestion
* User names and other account information is no longer considered to be sensitive data for the queries `py/clear-text-logging-sensitive-data` and `py/clear-text-storage-sensitive-data`, since this lead to many false positives.
```
An attempt at a slight reformulation.
